tutorials/shaders/advanced_postprocessing.rst

```diff
@@ -46,6 +46,8 @@ and sets the vertex position in clip space directly.
 .. code-block:: glsl
 
   shader_type spatial;
+  // Prevent the quad from being affected by lighting and fog. This also improves performance.
+  render_mode unshaded, disable_fog;
```

Comment:
While we're at it, shouldn't the `skip_vertex_transform` render mode also be used?

Reply:
Its not necessary, `POSITION` already skips the vertex transform

Reply:
cc @clayjohn